src/AbstractFixer.php

```diff
@@ -80,6 +86,63 @@ public function supports(\SplFileInfo $file)
         return true;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            @trigger_error(
+                'Passing NULL to set default configuration is deprecated and will not be supported in 3.0, use an empty array instead.',
+                E_USER_DEPRECATED
+            );
```

Comment:
for completeness, as discussed @ https://github.com/julienfalque/PHP-CS-Fixer/pull/1#discussion_r102440937 this deprecation is indeed valid and should stay in this PR :+1: